### ISSUE TITLE
Optional send diagnostics to datalayer.storage - for example if anon ip address can be collected, then we can build a map of all the DIG nodes to show how many are scattered across the world

### ISSUE THREAD

**Issue by MichaelTaylor3D** [Datalayer-Storage/Decentralized-Internet-Gateway#22]

**Comment by dkackman** [comment]
I have wired Microsoft Application Insights in but haven't really turned it on. 

https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview

They call it Azure monitoring but work with non-azure stuffs.

**Comment by dkackman** [comment]
Yeah, distributed telemetry is gonna be some work. I'm using apps insights for my node, but that entails setting a key specific to my azure account which isn't meant to be shared.
